calories/calories.rb

```diff
@@ -0,0 +1,84 @@
+require 'sqlite3'
+
+db = SQLite3::Database.new("daily_cal.db")
+db.results_as_hash = true
+
+continue = true
+$table_exists = false
```

Comment:
Avoid global variables at all times.  Once you start working with multiple files this is very dangerous and can expose your code in unanticipated ways.